apps/passport-client/components/screens/AddSubscriptionScreen.tsx

```diff
@@ -224,41 +357,109 @@ function AlreadySubscribed({
   subscriptions: FeedSubscriptionManager;
 }) {
   const onUnsubscribeClick = useCallback(() => {
-    if (
-      !window.confirm("are you sure you want to unsubscribe from this feed?")
-    ) {
-      return;
-    }
+    subscriptions.unsubscribe(existingSubscription.id);
+  }, [existingSubscription.id, subscriptions]);
 
-    subscriptions.unsubscribe(
-      existingSubscription.providerUrl,
-      existingSubscription.feed.id
-    );
-  }, [
-    existingSubscription.feed.id,
-    existingSubscription.providerUrl,
-    subscriptions
-  ]);
+  const options: Intl.DateTimeFormatOptions = {
+    weekday: "long",
+    year: "numeric",
+    month: "long",
+    day: "numeric"
+  };
 
   return (
     <div>
-      You're already subscribed to this feed! <br />
-      subscribed at{" "}
-      {new Date(existingSubscription.subscribedTimestamp).toISOString()}
-      <br />
+      You have been subscribed to this feed since{" "}
+      {new Date(existingSubscription.subscribedTimestamp).toLocaleDateString(
+        navigator.language,
+        options
+      )}
+      <Spacer h={8} />
       <Button onClick={onUnsubscribeClick} size="small" style="danger">
         Unsubscribe
       </Button>
     </div>
   );
 }
 
+function Spinner({ text, show }: { text: string; show: boolean }) {
+  return (
+    <SpinnerWithText>
+      {show && (
+        <SpinningSVG
```

Comment:
implement this svg the way the rest of the svgs in the app are implemented, and make sure it's appropriately licensed

Reply:
Hmm. The SVG itself may not be licensed for use. Replaced it with a similar one that is.